.github/workflows/pipeline.yaml

```diff
@@ -0,0 +1,24 @@
+name: Pipeline
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+    types: [opened, synchronize]
+jobs:
+  Lint:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Install node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+          cache: npm
+      - name: Install dependencies
+        run: npm install
+      - name: Lint code
+        run: npm run lint:fix
```

Comment:
"npx prettier . --write" will only modify files in a VM. Improve it by running "--check"